linode/manager
# M3-5756 Change feedback link

## Description

**What does this PR do?**
This changes the "Provide Feedback link in the footer from a mailto to an external link to the [feedback site](https://www.linode.com/feedback/).

I removed the `createMailto` function and related tests.

I also removed the other place `createMailto` was used, which was to send a report of a failed invoice generation to the feedback email, which I don't believe has _ever_ been used.


## How to test

**What are the steps to reproduce the issue or verify the changes?**
Verify clicking "Provide Feedback" in the footer opens a new tab containing the feedback site. 

Also look through the code for other mentions of the feedback email I may have missed. 

**How do I run relevant unit tests?**
